### PR TITLE
fix: install hadolint for hadolint command

### PR DESCRIPTION
**Pull request by brivu**
Currently if the `docker/hadolint` command is used with an `executor` other than the `hadolint` docker image, the  command fails. This `PR` installs hadolint if it isn't already.

### CODE REVIEW

File: src/scripts/hadolint.sh [hunk 1]
```diff
@@ -2,6 +2,34 @@
...
+	
+  if [ "${SYS_ENV_PLATFORM}" != "Darwin" ]; then
+    set -x
+    $SUDO wget -O /bin/hadolint "https://github.com/hadolint/hadolint/releases/latest/download/hadolint-${SYS_ENV_PLATFORM}"
+    $SUDO chmod +x /bin/hadolint
+    set +x
+  fi
+}
+
+if [ ! "$(command -v hadolint)" ]; then
```
Comment (EricRibeiro) [inline src/scripts/hadolint.sh:29]: You can avoid the subshell by invoking the command directly:

```suggestion
if ! command -v hadolint &> /dev/null; then
```

**Review by EricRibeiro — CHANGES_REQUESTED** [review]

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/docker?version=dev:a06236d8e3340e54b8c3c4fb9e95776084a35733

File: src/scripts/hadolint.sh [hunk 2]
```diff
@@ -2,6 +2,34 @@
 eval "$SCRIPT_UTILS"
 expand_env_vars_with_prefix "PARAM_"
 
+if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
+
+Install_Hadolint() {
+  if uname -a | grep "Darwin"; then
+    export SYS_ENV_PLATFORM="Darwin"
```
Comment (EricRibeiro) [inline src/scripts/hadolint.sh:9]: Are you using `SYS_ENV_PLATFORM` outside this script? If the answer is "no", I believe you don't to `export` the variable.

**Review by EricRibeiro — APPROVED** [review]

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/docker?version=dev:c738e3152f32245b721d8aad4fa33f3c08b54386

**Review by KyleTryon — APPROVED** [review]

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/docker?version=dev:c738e3152f32245b721d8aad4fa33f3c08b54386

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/docker?version=dev:fd6f8a79381e3f7e7603519341eefe854f8a5c2d

**Review by KyleTryon — APPROVED** [review]

**Comment by orb-publisher** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/circleci/docker?version=dev:b6600db35dd7b9b376f91b3e0fd27dcabc538125

**Pull request merged**